### ISSUE TITLE
trnc multi grid tests have errors on some platforms

### ISSUE THREAD

**Issue by JessicaMeixner-NOAA** [NOAA-EMC/WW3#1090]
**Describe the bug**
@ukmo-ccbunney is getting errors: 

https://github.com/NOAA-EMC/WW3/pull/1089#issuecomment-1746891912

```
Error occured during /home/d02/frey/WW3/REGS_GNU/regtests/mww3_test_02/work_PR3_UNO_a_c/exe/ww3_trnc execution
../../bin/matrix10.out:ERROR: Error occured during /home/d02/frey/WW3/REGS_GNU/regtests/mww3_test_02/work_PR3_UQ_a/exe/ww3_trnc execution
../../bin/matrix10.out:ERROR: Error occured during /home/d02/frey/WW3/REGS_GNU/regtests/mww3_test_02/work_PR3_UQ_a_c/exe/ww3_trnc execution
```

I don't see these tests being run at all.    Need to determine why there's bugs for @ukmo-ccbunney and why perhaps we aren't even trying to run these at EMC.

**Comment by JessicaMeixner-NOAA** [comment]
For NCEP the issue is we use output all and there's not an option for that for the track output.

**Comment by ukmo-ccbunney** [comment]
Ah - that explains it. I can add that as a fix in #1091 if you like?
